buildlib/cbuild

```diff
@@ -170,6 +170,41 @@ class centos7_epel(centos7):
         res.lines.append("RUN ln -s /usr/bin/cmake3 /usr/local/bin/cmake && ln -sf /usr/bin/python3.4 /usr/bin/python3");
         return res;
 
+class amazonlinux1(YumEnvironment):
+    docker_parent = "amazonlinux:1";
+    pkgs = {
+        'cmake',
+        'gcc',
+        'libnl3-devel',
+        'libudev-devel',
+        'make',
+        'pkgconfig',
+        'python',
+        'python-argparse',
+        'python-docutils',
+        'python27-docutils',
```

Comment:
Why both?

Reply:
`python-docutils` didn't work so I added the python27 one. I'll test without the former one and change accordingly.

Reply:
Removed `python-docutils`.